include/rabbit_peer_discovery_etcd.hrl

```diff
@@ -1,46 +1,18 @@
--define(CONFIG_MODULE, rabbit_peer_discovery_config).
--define(UTIL_MODULE,   rabbit_peer_discovery_util).
--define(HTTPC_MODULE,  rabbit_peer_discovery_httpc).
-
 -define(BACKEND_CONFIG_KEY, peer_discovery_etcd).
 
-
--define(CONFIG_MAPPING,
-         #{
-          etcd_scheme    => #peer_discovery_config_entry_meta{
-                              type          = string,
-                              env_variable  = "ETCD_SCHEME",
```

Comment:
This drops support for environment variable configuration. We retained it originally for better `rabbitmq-autocluster` migration experience but never recommended it has any reasonably complex configuration via env variables ends up being too painful to use or troubleshoot. The docs have been [saying as much](https://www.rabbitmq.com/cluster-formation.html#configuring) since peer discovery introduction in 3.7.0.

Reply:
Several things would be awkward to configure via env variables, e.g. endpoint lists or TLS settings. Passwords, too: this options would not support encryption like RabbitMQ `advanced.config` file does.

Reply:
No. I don't think it's very relevant. There will be a changelog note. We've been recommending against env variable usage for peer discovery settings for several years now.